### PR TITLE
Some path normalization

### PR DESCRIPTION
**Pull request by blinkdog**
**What does this PR do:**
Normalizes a few paths in nine different source files.

**Changelog:**
:cl:
fix: Normalized some paths in the code tree
/:cl:

### CODE REVIEW

**Comment by variableundefined** [comment]
Does not compile.

**Comment by taukausanake** [comment]
@blinkdog you forgot to check your tabbing when you gave direct paths

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -1407,27 +1407,27 @@ var/list/ghostteleportlocs = list()
 	dynamic_lighting = 0
 	valid_territory = FALSE
 
-	auxport
+/area/solar/auxport
 		name = "\improper Fore Port Solar Array"
 		icon_state = "panelsA"
```
Comment (taukausanake) [inline code/game/area/Space Station 13 areas.dm]: Right here you removed a tab and replaced it with the direct path but you didn't remove the tabs of the variables below it. Repeat for other declarations below

Comment (blinkdog) [reply code/game/area/Space Station 13 areas.dm]: Ack, that's not good. I'll follow up on this after work.

**Comment by blinkdog** [comment]
After outdenting, I was able to compile this with BYOND 512.1462:
```
pmeade@konoha:~/data/Paradise$ BYOND_MAJOR=512 BYOND_MINOR=1462 dmake rebuild
Cleaning project...
Rebuilding project...
paradise.dmb - 0 errors, 0 warnings (1/24/19 4:38 pm)
Total time: 1:18
```

**Pull request merged**